gradio/components/number.py

```diff
@@ -79,10 +80,12 @@ def __init__(
             precision: Precision to round input/output to. If set to 0, will round to nearest integer and convert type to int. If None, no rounding happens.
             minimum: Minimum value. Only applied when component is used as an input. If a user provides a smaller value, a gr.Error exception is raised by the backend.
             maximum: Maximum value. Only applied when component is used as an input. If a user provides a larger value, a gr.Error exception is raised by the backend.
+            step: The interval between numbers in the input field. If None, a default of 1 is set.
```

Comment:
Nit: updating the wording a little bit based on the change above.
```suggestion
            step: The interval between allowed numbers in the component. Can be used along with `minimum` and `maximum` to create a range of legal values.
```